schema_registry/client/client.py

```diff
@@ -19,39 +21,35 @@
 logger = logging.getLogger(__name__)
 
 
-def get_response_and_status_code(response: httpx.Response) -> tuple:
-    """
-    Returns a tuple containing response json and status code
+def get_response_and_status_code(response: httpx.Response) -> typing.Tuple[typing.Any, int]:
+    """Returns a tuple containing response json and status code.
 
     Args:
-        response (httpx.Response): Http response object
+        response: Http response object
 
     Returns:
-       tuple(dict, int)
+       A tuple of the JSON response and the status code
     """
-
     return (
         response.json(),
         response.status_code,
     )
 
 
 class BaseClient:
-    """
-    A client that talks to a Schema Registry over HTTP
+    """A client that talks to a Schema Registry over HTTP.
 
     Attributes:
-        url str | typing.Dict: Url to schema registry or dictionary containing client configuration.
```

Comment:
Are the types are proper generated in the documentation?

Reply:
I verified and noticed an issue on Class declaration, now fixed it:

![screen](https://github.com/marcosschroh/python-schema-registry-client/assets/1858751/f95f88f2-2d1a-487f-8e6e-e8cbe9fe0cf9)

From what I see everything looks good !